tests/e2e/specs/user-input-questions.test.js

```diff
@@ -76,7 +76,9 @@ describe( 'User Input Settings', () => {
 			'enter keywords',
 			async () => {
 				await expect( page ).toClick( '.googlesitekit-user-input__buttons--next' );
-				await expect( page ).toFill( '#searchTerms-keyword-0', 'One,Two,Three,' );
+				await expect( page ).toFill( '#searchTerms-keyword-0', 'One' );
+				await expect( page ).toFill( '#searchTerms-keyword-1', 'Two' );
+				await expect( page ).toFill( '#searchTerms-keyword-2', 'Three' );
```

Comment:
I think this is likely the only additional change needed on the original PR.

Reply:
Not the only one, see my comment above. Try to revert `allowEmptyValues` changes and comment out one of these lines. Tests will start failing.